### PR TITLE
feature: add firecrawl docker compose

### PR DESCRIPTION
**Pull request by densumesh**

### CODE REVIEW

File: docker-compose-firecrawl.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      - PORT=3000
+      - PROXY_SERVER=${PROXY_SERVER}
+      - PROXY_USERNAME=${PROXY_USERNAME}
+      - PROXY_PASSWORD=${PROXY_PASSWORD}
+      - BLOCK_MEDIA=${BLOCK_MEDIA}
+    networks:
+      - backend
+
+  firecrawl-api:
+    image: trieve/firecrawl:latest
```
Comment (cdxker) [inline docker-compose-firecrawl.yml:16]: Are we making a custom firecrawl image?

Comment (densumesh) [reply docker-compose-firecrawl.yml:16]: yes already made it

Comment (cdxker) [reply docker-compose-firecrawl.yml:16]: Ah gotcha, its just firecrawl hasn't published one.

**Review by cdxker — APPROVED** [review]

**Comment by cdxker** [comment]
Looks great, will push to make this a glasskube package.

**Pull request merged**